pkg/stub/config.go

```diff
@@ -286,19 +281,24 @@ func (h *Handler) ProcessManagementField(cfg *v1.Config) (bool, bool, error) {
 			if err != nil {
 				return false, true, h.processError(cfg, v1.SamplesExist, corev1.ConditionUnknown, err, "The error %v during openshift namespace cleanup has left the samples in an unknown state")
 			}
-			// explicitly reset samples exist and import cred to false since the Config has not
-			// actually been deleted; secret watch ignores events when samples resource is in removed state
+			// explicitly reset exist/inprogress/error to false
 			now := kapis.Now()
-			condition := util.Condition(cfg, v1.SamplesExist)
-			condition.LastTransitionTime = now
-			condition.LastUpdateTime = now
-			condition.Status = corev1.ConditionFalse
-			util.ConditionUpdate(cfg, condition)
+			conditionsToSet := []v1.ConfigConditionType{v1.SamplesExist, v1.ImageChangesInProgress, v1.ImportImageErrorsExist}
+			for _, c := range conditionsToSet {
+				condition := util.Condition(cfg, c)
+				condition.LastTransitionTime = now
```

Comment:
Not sure I follow your question @adambkaplan .... are you asking if it is valid to make the transition?

If so I would say yes.

Or are you asking if we can update multiple conditions at the same time?

If so again I would say yes.

Otherwise, please clarify.